pkg/util/httpstream/spdy/roundtripper_test.go

```diff
@@ -146,23 +264,56 @@ func TestRoundTripAndNewConnection(t *testing.T) {
 			stream := <-streamCh
 			io.Copy(stream, stream)
 		}))
-		// TODO: Uncomment when fix #19254
-		// defer server.Close()
+		defer server.Close()
```

Comment:
yeah, https://github.com/kubernetes/kubernetes/issues/19254#issuecomment-174734626 was closed not with a fix, but by commenting out the .Close() calls